File: 코딩 기초 트레이닝/접미사인지 확인하기-p181908/접미사인지 확인하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181908

def solution(my_string, is_suffix):
    if is_suffix in my_string:
        if len(my_string) - len(is_suffix) == my_string.rfind(is_suffix):
            return 1
        else:
            return 0
    else:
        return 0

File: 코딩 기초 트레이닝/접미사인지 확인하기-p181908/접미사인지 확인하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181908

function solution(my_string, is_suffix) {
    if(my_string.indexOf(is_suffix) < 0) {
        return 0;
    } else {
        if(my_string.lastIndexOf(is_suffix) === my_string.length - is_suffix.length) {
            return 1;
        } else {
            return 0;
        }
    }
}
